astropy/timeseries/tests/test_downsample.py

```diff
@@ -6,12 +6,17 @@
 
 from astropy import units as u
 from astropy.time import Time
+from astropy.utils.exceptions import AstropyUserWarning
 
 from astropy.timeseries.sampled import TimeSeries
 from astropy.timeseries.downsample import aggregate_downsample, reduceat
 
+
 INPUT_TIME = Time(['2016-03-22T12:30:31', '2016-03-22T12:30:32',
-                   '2016-03-22T12:30:33', '2016-03-22T12:30:34'])
+                   '2016-03-22T12:30:33', '2016-03-22T12:30:34',
+                   '2016-03-22T12:30:35'])
+ts = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4, 5]], names=['a'])
+ts_units = TimeSeries(time=INPUT_TIME, data=[[1, 2, 3, 4, 5] * u.count], names=['a'])
```

Comment:
```suggestion
```
Ah, these apparently need to be moved inside the function; see files changes in #11638!